packages/git/git.pacscript

```diff
@@ -20,5 +20,6 @@ build() {
 }
 
 install() {
-          sudo make install DESTDIR=/usr/src/pacstall/git
+          sudo make install DESTDIR=/usr/src/pacstall/$name
+          sudo mv COPYING /usr/src/pacstall/git/usr/share/licenses/$name            
 }
```

Comment:
```suggestion
}

removescript() {
    sudo rm -rf /usr/share/licenses/$name
}
```
Removes the empty directory after uninstall.

Reply:
Done